packages/gitbook/src/components/DocumentView/CodeBlock/highlight.ts

```diff
@@ -81,18 +81,32 @@ export async function highlight(block: DocumentBlockCode): Promise<HighlightLine
     });
 }
 
+const syntaxAliases: Record<string, BundledLanguage> = {
+    // "Parser" language does not exist in Shiki, but it's used in GitBook
+    // The closest language is "Blade"
+    parser: 'blade',
+};
+
+function checkIsBundledLanguage(lang: string): lang is BundledLanguage {
+    return lang in bundledLanguages;
+}
+
 /**
  * Validate a language name.
  */
-function getLanguageForSyntax(syntax: string): keyof typeof bundledLanguages | null {
-    // @ts-ignore
-    const lang = bundledLanguages[syntax];
-    if (!lang) {
-        return null;
+function getLanguageForSyntax(syntax: string): BundledLanguage | null {
+    // Check if the syntax is a bundled language.
```

Comment:
```suggestion
    syntax = syntax.toLowerCase();
    // Check if the syntax is a bundled language.
```

Should you normalize it?